lib/collection/src/shards/local_shard/mod.rs

```diff
@@ -101,7 +101,6 @@ pub struct LocalShard {
     pub(super) search_runtime: Handle,
     disk_usage_watcher: DiskUsageWatcher,
     read_rate_limiter: ParkingMutex<Option<RateLimiter>>,
-    write_rate_limiter: ParkingMutex<Option<RateLimiter>>,
```

Comment:
Is there a specific reason why we only move `write_rate_limiter` and not both?

Reply:
I will move the read limiter as well if it is required to fix potential miscounts.

I also prefer splitting my work in small PRs :smile_cat: